chatapplication/src/components/Searcharea.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './searcharea.css';
+
+
+const STYLES = "search-area"
+
+export const Searcharea = ({ 
+    type, 
+    onClick, 
+    searchStyle,
+    
+}) => {
+
+    const checkSearchStyle = STYLES.includes(searchStyle) ? searchStyle : STYLES[0];
+    return (
+        <div className='search-area-div'>
+            <div className={`searcharea ${checkSearchStyle}`}onClick={onClick} type={type} action="/url" method="GET">
```

Comment:
what is this `check` in front of the `searchStyle`?
If I am a new member in your group, am I going to understand what is happening here?